nebullvm/tools/base.py

```diff
@@ -1,36 +1,52 @@
 from dataclasses import dataclass
 from enum import Enum
-from typing import Tuple, List, Dict, Union
+from typing import Tuple, List, Union, Dict
+
+
+class QuantizationType(Enum):
+    DYNAMIC = "DYNAMIC"
+    STATIC = "STATIC"
+    QAT = "QAT"
```

Comment:
I think we can safely remove it. QAT will be added as a compression technique in the future.